app/assets/javascripts/discourse-common/addon/config/environment.js

```diff
@@ -13,7 +13,7 @@ export function setEnvironment(e) {
 }
 
 export function isTesting() {
-  return environment === "testing";
+  return environment === "testing" || environment === "test";
```

Comment:
system tests are using Rails environment which is `test`

Reply:
LOL disregard my other comment, this is nice :)